src/main/java/teammates/ui/controller/InstructorCommentsPageAction.java

```diff
@@ -52,13 +52,10 @@ public ActionResult execute() throws EntityDoesNotExistException {
         
         List<String> coursePaginationList = new ArrayList<String>(); 
         String courseName = getCoursePaginationList(coursePaginationList);
-        
         data = new InstructorCommentsPageData(account);
-        data.isViewingDraft = isViewingDraft;
-        data.currentInstructor = instructor;
-        data.isDisplayArchive = isDisplayArchivedCourse;
-        data.courseId = courseId;
-        data.courseName = courseName;
+        
```

Comment:
Is it possible to move the variables in this class into the method scope rather than the class scope?